src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/LeggTilEllerFjernBrevmottakere/BrevmottakerSkjema.tsx

```diff
@@ -24,7 +25,11 @@ interface Props {
 }
 
 const BrevmottakerSkjema = ({ erLesevisning, skjema, navnErPreutfylt }: Props) => {
-    const gyldigeMottakerTyper = Object.values(Mottaker);
+    const erPåDokumentutsending = useLocation().pathname.includes('dokumentutsending');
```

Comment:
Betyr dette at man sender ut på fagsaksnivå? Kunne det isåfall ha kommet tydelig frem i variabelnavnet?

Reply:
Ja, hva med f.eks `erPåDokumentutsendingForFagsak`?

Reply:
2ef1df0041d5af9469dbcc0a3301ddd04a012363